liferay-workspace/tests/build.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'groovy'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'org.codehaus.groovy:groovy-all:2.4.4'
+    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+	testCompile gradleTestKit()
+}
+
+//check.dependsOn ':blade-tests:initBundle'
```

Comment:
We need to make gradle call call initBundle on the root project first, we don't need to call initBundle inside of the groovy test file.